### PR TITLE
Style fixes

### PR DESCRIPTION
**Pull request by fuzzythecat**

### CODE REVIEW

File: keras_applications/densenet.py [hunk 1]
```diff
@@ -179,7 +179,7 @@ def DenseNet(blocks,
                          'or the path to the weights file to be loaded.')
 
     if weights == 'imagenet' and include_top and classes != 1000:
-        raise ValueError('If using `weights` as imagenet with `include_top`'
+        raise ValueError('If using `weights` as `imagenet` with `include_top`'
```
Comment (taehoonlee) [inline keras_applications/densenet.py]: `imagenet` -> `'imagenet'`

File: keras_applications/inception_v3.py [hunk 2]
```diff
@@ -372,7 +372,7 @@ def InceptionV3(include_top=True,
     # Create model.
     model = models.Model(inputs, x, name='inception_v3')
 
-    # load weights
+    # Load weights
```
Comment (taehoonlee) [inline keras_applications/inception_v3.py]: Load weights -> Load weights.

**Comment by fuzzythecat** [comment]
@taehoonlee Made the changes.

File: keras_applications/densenet.py [hunk 3]
```diff
@@ -179,7 +179,7 @@ def DenseNet(blocks,
                          'or the path to the weights file to be loaded.')
 
     if weights == 'imagenet' and include_top and classes != 1000:
-        raise ValueError('If using `weights` as imagenet with `include_top`'
+        raise ValueError('If using `weights` as \'imagenet\' with `include_top`'
```
Comment (taehoonlee) [inline keras_applications/densenet.py]: I mean \`'imagenet'\` not \\'imagenet\\'. Sorry for the confusion.

Comment (taehoonlee) [reply keras_applications/densenet.py]: Please see an [example](https://github.com/keras-team/keras-applications/blob/master/keras_applications/inception_resnet_v2.py#L121).

Comment (fuzzythecat) [reply keras_applications/densenet.py]: Oh, I misunderstood. Got it now.

**Comment by fuzzythecat** [comment]
@taehoonlee Used double quotation marks instead to avoid syntax error.

**Comment by taehoonlee** [comment]
Thank you @fuzzythecat, LGTM.

**Pull request merged**